backend/Contexture.Api.Tests/Tests.fs

```diff
@@ -2,16 +2,17 @@ module Tests
 
 open Contexture.Api
 open Xunit
+open Database
 
 [<Fact>]
 let ``Unversioned JSON deserialization`` () =
-    let exampleInputPath = "../../../../example/restaurant-db.json"
+    let exampleInputPath = @"../../../../../example/restaurant-db.json"
```

Comment:
should we add the `restaurant-db.json` as linked resource to the test-project (with copy to output directory)?

Reply:
Since we won't be moving the folders around in the near future, it won't break. 😛 I want to get rid of this test anyway before this is merged in since it's just there for development. Or replace it with a meaningful set of tests.